Helper/Config.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/*
+ * Copyright (c) Aligent Consulting. All rights reserved.
+ */
+
+declare(strict_types=1);
+namespace Aligent\PrerenderIo\Helper;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+
+class Config
+{
+    private const XML_PATH_RECACHE_ENABLED = 'system/prerender_io/enabled';
+    private const XML_PATH_PRERENDER_TOKEN = 'system/prerender_io/token';
+
+    /** @var ScopeConfigInterface  */
```

Comment:
Is there a reason to have these docblocks I'm not aware of?

Reply:
Coding standards pipeline requires this.

Reply:
Ah I can see that now. Kinda assumed they'd drop that rule since it now requires PHP 7 but anyways...